'Because of this' semantic discrepancy (#12059)
istio/istio.io: content/en/docs/concepts/traffic-management/index.md

Routing rules are **evaluated in sequential order from top to bottom**, with the
first rule in the virtual service definition being given highest priority. In
this case you want anything that doesn't match the first routing rule to go to a

> default destination, specified in the second rule. But, in the second
> rule has no match conditions and just directs traffic to the v3 subset.

Reviewer: I'm not sure that this sounds correct either. Maybe 
```suggestion
default destination, specify that in a second rule which has no match conditions.
Here, the rule directs traffic to the v3 subset.
```

Author: This sounds good
